backend/src/lib/utilities/index.js

```diff
@@ -0,0 +1,13 @@
+import envVarsSchema from '../constants/env-vars-schema'
```

Comment:
I don't have a hugely strong opinion on this - but wonder whether we might be better served to have a `utils/` folder, into which we put something like an `EnvVar` class with the method `get`

Purely thinking it might be more maintainable to have lots of `utils/theThing`s rather than one single utility file that's used for lots of different things (and could grow in quite diffuse ways)

Reply:
@tomajwinter: We now have a [`backend/src/utils/`](https://github.com/kyan/jukebox-js/tree/master/backend/src/utils) folder, and I've now created [`EnvVars`](https://github.com/kyan/jukebox-js/tree/get-env-var/backend/src/utils/env-vars) within it :thumbsup: